File: javascript/leetcode/ContainsDuplicate.js
function containsDuplicate(nums) {
    let newArray = new Set();
    for (let i = 0; i < nums.length; i++) {
        if (newArray.has(nums[i])) {
            return true;
        }
        newArray.add(nums[i]);
    }
    return false;
}

File: java/leetcode/easyLevel/ContainsDuplicate.java
package easyLevel;
import java.util.HashSet;

class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> array = new HashSet(); 

        for(int i = 0; i < nums.length; i++) {
            if(array.contains(nums[i])) {
                return true; 
            } 
            array.add(nums[i]);
        }
        return false;
    }
}